systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture_thumbnail.url }}" class="profile-pic img-square img-responsive crop_img" alt="{{ user }} profile picture"/>
+  </a>
+{% else %}
+  <a href="{{ user.get_absolute_url }}">      
```

Comment:
Using text-center(bootstrap class same as text-align: center;) is setting text to the right of image - http://gyazo.com/67754ad9ea4d4423572c4b089359bdf0